browser/ui/views/location_bar/brave_location_bar_view.cc

```diff
@@ -36,6 +33,12 @@ void BraveLocationBarView::Update(const content::WebContents* contents) {
   LocationBarView::Update(contents);
 }
 
+void BraveLocationBarView::UpdateBookmarkStarVisibility() {
+  if (star_view_) {
+    star_view_->SetVisible(false);
```

Comment:
Much clear than before!

How about adding a browser test to check |star_view_| is always invisible?